go-backend/db/user_test.go

```diff
@@ -0,0 +1,74 @@
+package db
```

Comment:
@thanekarpooja27 lets remove this integration test file, we don't need this anymore. Instead we will be writing test cases using https://github.com/DATA-DOG/go-sqlmock

But we can do that in a separate PR.

Reply:
only the db test case file